projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.spec.ts

```diff
@@ -92,21 +92,17 @@ describe('TabParagraphContainerComponent', () => {
       of(mockTabComponentData2),
       of(mockTabComponentData3)
     );
-    let childComponents$: Observable<any>[];
+    let childComponents: any[];
     component.components$
-      .subscribe(components => (childComponents$ = components))
+      .subscribe(components => (childComponents = components))
       .unsubscribe();
 
-    for (let i = 0; i < childComponents$.length; i++) {
-      childComponents$[i]
-        .subscribe(tab => {
-          expect(tab).toEqual({
-            flexType: mockComponents[i],
-            uid: mockComponents[i],
-            title: `productTabs.${mockComponents[i]}`,
-          });
-        })
-        .unsubscribe();
+    for (let i = 0; i < childComponents.length; i++) {
+      expect(childComponents[i]).toEqual({
+        flexType: mockComponents[i],
+        uid: mockComponents[i],
+        title: `productTabs.${mockComponents[i]}`,
```

Comment:
The component is almost generic, expect for this `productTabs` prefix. Can we make it `CMSTabParagraphContainer.tabs.`? or something similar?